src/ert/_c_wrappers/job_queue/driver.py

```diff
@@ -53,15 +57,8 @@ def __init__(self, driver_type, max_running=1, options=None):
                 self.set_option(key, value)
         self.set_max_running(max_running)
 
-    def set_option(self, option, value):
-        """
-        Set the driver option @option to @value.
```

Comment:
Maybe keeping the first line makes sense

Reply:
Hmm, I removed it because it does not give any extra information compared to the function name and signature.